tests/NSubstitute.Acceptance.Specs/PerfTests.cs

```diff
@@ -85,6 +85,18 @@ public void Muiltiple_return_configurations_dont_lead_to_memory_leak()
             }
         }
 
+        [Test]
+        public void Delegate_proxy_container_is_cached()
```

Comment:
Did this test fail before the change?
I'm not 100% sure this is checking the caching.